### PR TITLE
From Bash to Gulp to SCSS to CSS to HTML

### PR DESCRIPTION
**Pull request by Nickstr**
mgrn

to run: bash raffler ,.-~*'¨¯¨'*•~-.¸-(_	filename_)-,.-~*'¨¯¨'*•~-.¸

### CODE REVIEW

File: nickspelt-html/raffler [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+if [ ! -d node_modules ];then
+    npm install
+fi
+
+# Random name script from rjkip
+# https://github.com/domcode/rafflers/tree/master/rjkip-bash
+NAME=$(head -$((${RANDOM} % `wc -l < $1` + 1)) $1 | tail -1)
+
+# Pass the winners name Gulp
+gulp css --winner "$NAME"
```
Comment (wjzijderveld) [inline nickspelt-html/raffler]: Only works when globally installed
Use `node ./node_modules/gulp/bin/gulp` instead (still assumes node installed, but that should be handled in install.sh)

File: nickspelt-html/raffler [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Open the page
+open index.html
```
Comment (wjzijderveld) [inline nickspelt-html/raffler]: Does not work on linux :(

`xdg-open` works on most distro's, but probably won't work on a Mac...

File: nickspelt-html/raffler (hunk 1 above)
Comment (wjzijderveld) [inline nickspelt-html/raffler]: Sorry, should be `node ./node_modules/gulp/bin/gulp.js`

**Comment by wjzijderveld** [comment]
Done for production:

https://s3-eu-west-1.amazonaws.com/wjzijderveld/domcode/domcode_raffler.mp4

**Comment by rosstuck** [comment]
rube goldberg machine

**Pull request merged**